### ISSUE TITLE
Update Marchantaria Use Case notebook to work with restored API

### ISSUE THREAD

**Issue by emiliom** [ODM2/ODM2RESTfulWebServices#7]
[ODM2RESTdemo_MarchantariaUseCase.ipynb](https://github.com/BiG-CZ/BiG-CZ-Toolbox/blob/master/ipynotebooks/ODM2RESTdemo_MarchantariaUseCase.ipynb) Jupyter notebook was **last updated 2015-Aug-9**. AFAIK, it's still the best public client demo available of the ODM2 REST API.

Now that the supporting [ODM2 REST API backend for the Marchantaria use case has been restored](6) (now hosted on a UW AWS instance at http://54.213.227.245:8080/docs/), a next logical step is to update the notebook so it runs fine against this new REST API instance. Right now it's not working. My guess is that the notebook was written to the first release of the API, but when a [new version was released later in August 2015 based on my feedback](1), I never updated the notebook to the changed interface. *This will be the first possibility I'll investigate to debug the notebook.*

cc @lsetiawan

**Comment by emiliom** [comment]
I've made progress getting valid responses!! Woo-hoo. And yes, the requests need to be adjusted to the changed API.

I'm currently struggling with getting the `RelatedSamplingFeatures` to extract the `SamplingFeatures` and `Sites` information, but I'll get there.

**Comment by lsetiawan** [comment]
Woo hoo! That sounds awesome! :) I'm glad to hear that. Excited to see how that'll go.

**Comment by emiliom** [comment]
> I'm glad to hear that.

I figured you'd enjoyed hearing about it :stuck_out_tongue_winking_eye:

**Comment by emiliom** [comment]
Alrighty, I've got almost everything running again in the notebook! The only exception is the folium map, but I may be using an old approach that no longer works. I'll look up newer examples.

Also, there seem to be instabilities in the service, where sometimes the same request returns an error, and sometimes it doesn't. No big deal for a prototype.

I'll upload the new notebook to github once I've cleaned it up and fixed the folium map.
